pkg/graveler/errors.go

```diff
@@ -17,6 +17,8 @@ var (
 	ErrNotUnique                    = wrapError(ErrUserVisible, "not unique")
 	ErrPreconditionFailed           = errors.New("precondition failed")
 	ErrWriteToProtectedBranch       = wrapError(ErrUserVisible, "cannot write to protected branch")
+	ErrCast                         = errors.New("cannot cast value")
+	ErrNilValue                     = errors.New("nil value")
```

Comment:
It feels like we're reimplementing golang errors. We would always send them when we fail to parse Protobuf, right? So why not `ErrReadingProtobuf`?

Reply:
Another (better IMO) option `ErrReadingFromStore`

Reply:
done